slf4j-bridge/src/main/scala-2/org/slf4j/impl/ZioLogger.scala

```diff
@@ -4,6 +4,9 @@ import org.slf4j.helpers.{ MarkerIgnoringBase, MessageFormatter }
 import zio.{ Cause, ZIO }
 
 class ZioLogger(name: String, factory: ZioLoggerFactory) extends MarkerIgnoringBase {
+
+  override def getName: String = name
```

Comment:
There's already `protected String name` in `org.slf4j.helpers.NamedLoggerBase` but Scala doesn't allow implementing it with `override val`, so that I directly overrode `getName` here.

Reply:
i think also something like:

```scala
class ZioLogger(n: String, factory: ZioLoggerFactory) extends MarkerIgnoringBase {
  this.name = n
  
}
```

could work

Reply:
Yes that will work. It's been too long since I wrote mutable Scala code :stuck_out_tongue: 

Would you prefer this to introducing intermediate abstract class as in 5b719d1a986108daa0c9a9585777f5a7896f71b8?

Reply:
solution with ZioLoggerBase is probably nicer, I am ok with that